client/src/pages/Home.js

```diff
@@ -8,12 +8,16 @@ import "./Home.css";
 export function Home() {
 	const [mostRecentEnergizers, setMostRecentEnergizers] = useState([]);
 	const [ratedEnergizers, setRatedEnergizers] = useState([]);
+	const [initialRecentEnergizers, setInitialRecentEnergizers] = useState([]);
+	const [initialRatedEnergizers, setInitialRatedEnergizers] = useState([]);
 	useEffect(() => {
 		fetch("/api/energizers?sort_by=recent")
 			.then((response) => response.json())
 			.then((energizers) => {
 				const mostRecentEnergizers = energizers.slice(0, 2);
 				setMostRecentEnergizers(mostRecentEnergizers);
+				setInitialRecentEnergizers(mostRecentEnergizers);
```

Comment:
Same question - why is this data duplicated into two different states? It seems the only reason is that one state is being trimmed but that could easily happen when the state is being used.

Reply:
I've been updated query, so should be more clearly now , it's working!